CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -151,7 +159,7 @@ local function CreateTopBar()
 	function this:UpdateBackgroundTransparency()
 		local playerGui = Player:FindFirstChild('PlayerGui')
 		if playerGui then
-			topbarContainer.BackgroundTransparency = playerGui:GetTopbarTransparency()
+			topbarContainer.BackgroundTransparency = (not topbarEnabled and 1) or playerGui:GetTopbarTransparency()
```

Comment:
Now that we have ComputeTransparency lets move this logic into that function as well. This will make it less likely to run into some weird cases.